src/test/python_tests/test_path_specialization.py

```diff
@@ -91,7 +91,6 @@ def _handler(_params):
 
 
 def test_interpreter():
-    """Test linting using specific python path."""
```

Comment:
include this text

Reply:
Hello @karthiknadig,
Could you please guide me which text should I add because the function def _handler(_params): is already present at line 109?
Thank you

Reply:
You have removed: `"""Test linting using specific python path."""` that was under `def test_interpreter():`.

Reply:
Sorry. I have added back the comment. 
Could you please check once?